### ISSUE TITLE
plot_composite_network

### ISSUE THREAD

**Issue by lacpromoter** [baryshnikova-lab/safepy#3]
Is there an option to save the composite_network in pdf? Thanks!

**Comment by abarysh** [comment]
Yep, it's the save_fig argument (added in commit 6dade7384fa93776bcc32ec41bc2818be0d6b8e4). Example:

`sf.plot_composite_network(show_each_domain=False, save_fig='./output.pdf')
`

BTW, we're in the process of producing a more formal documentation for all the functions and arguments, so all of these functionalities should be a bit more transparent.
